Source/ChocolateyGui/ViewModels/SettingsViewModel.cs

```diff
@@ -270,6 +272,26 @@ public void Back()
 
             var sources = await _packageService.GetSources();
             Sources.AddRange(sources);
+        }
+
+        private void LocalizateFeaturesDescription(ChocolateyFeature feture)
+        {
+            var key = string.Format("SettingsViewModel_FeatureDescription_{0}", feture.Name);
+            var description = Resources.ResourceManager.GetString(key, Resources.Culture);
+            if (!string.IsNullOrWhiteSpace(description))
+            {
+                feture.Description = description;

```

Comment:
@mwallner yes, this is definitely something that can be done.  Personally, I am happy to do this, but last time I checked, there were some un related warnings in the build step, which would ultimately cause the build to fail.  If everyone is happy to proceed with warnings as errors, then this can certainly be enabled.